src/cpp/src/llm_pipeline.cpp

```diff
@@ -293,15 +678,23 @@ ov::genai::LLMPipeline::LLMPipeline(
     const std::string& device,
     const ov::AnyMap& plugin_config
 ) {
-    m_pimpl = make_unique<LLMPipelineImpl>(std::filesystem::path(model_path), tokenizer, device, plugin_config);
+    if (device == "NPU") {
+        m_pimpl = make_unique<NPULLMPipelineImpl>(std::filesystem::path(model_path), tokenizer, plugin_config);
+    } else {
+        m_pimpl = make_unique<LLMPipelineImpl>(std::filesystem::path(model_path), tokenizer, device, plugin_config);
+    }
 }
 
 ov::genai::LLMPipeline::LLMPipeline(
-    const std::string& path, 
-    const std::string& device, 
+    const std::string& path,
+    const std::string& device,
     const ov::AnyMap& config
 ) {
-    m_pimpl = make_unique<LLMPipelineImpl>(std::filesystem::path(path), device, config);
+    if (device == "NPU") {
```

Comment:
Can you handle meta devices? For example, will `MULTI:NPU` work?

Reply:
I suppose MULTI and LLM are not very compatible right now

Reply:
nah, there is not much sense as `NPUW` already proxy plugin